media/js/externalpages/pocket/mobile-nav.es6.js

```diff
@@ -21,7 +21,7 @@ function detectClickOutside(event) {
 
 function handleMenuOpen() {
     mobileNavWrapper.classList.add('active');
-    mobileNav.setAttribute('aria-expanded', true);
+    navOpenBtn.setAttribute('aria-expanded', true);
```

Comment:
Adjusting aria-expanded use here because it should go on the element controlling the display of more content
https://inclusive-components.design/menus-menu-buttons/#navigationmenubuttons